det3d/datasets/nuscenes/nusc_common.py

```diff
@@ -443,7 +443,7 @@ def _fill_trainval_infos(nusc, train_scenes, val_scenes, test=False, nsweeps=10,
 
                 # Get past pose
                 current_pose_rec = nusc.get("ego_pose", curr_sd_rec["ego_pose_token"])
-                global_from_car = transform_matrix(
+                curr_car_SE3_global = transform_matrix(
```

Comment:
could we rename this as `egovehicle_SE3_city` or `egovehicle_SE3_world` or `egoTw`? (shortest is preferred)

Reply:
Addressed in [this commit](https://github.com/hchittanuru3/CenterPoint/pull/3/commits/209cddf843a8962039acaffa9269c5a1a2ad58a7)